src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts

```diff
@@ -84,11 +84,15 @@ export const useWizardStore = defineStore("wizard", {
         // TODO convert from Components.Schemas.Transcript[] -> {[id: string]: Components.Schemas.Transcript}
         [wizard.steps.education.form.inputs.educationList.id]: transcriptsDict,
 
-        // TODO Character reference characterReference data from draft application
-        [wizard.steps.characterReference.form.inputs.firstName.id]: "",
-        [wizard.steps.characterReference.form.inputs.lastName.id]: "",
-        [wizard.steps.characterReference.form.inputs.email.id]: "",
-        [wizard.steps.characterReference.form.inputs.phoneNumber.id]: "",
+        // Character reference characterReference data from draft application
+        [wizard.steps.characterReferences.form.inputs.firstName.id]:
+          draftApplication.characterReferences && draftApplication.characterReferences[0] ? draftApplication.characterReferences[0].firstName : null,
+        [wizard.steps.characterReferences.form.inputs.lastName.id]:
+          draftApplication.characterReferences && draftApplication.characterReferences[0] ? draftApplication.characterReferences[0].lastName : null,
+        [wizard.steps.characterReferences.form.inputs.email.id]:
+          draftApplication.characterReferences && draftApplication.characterReferences[0] ? draftApplication.characterReferences[0].emailAddress : null,
+        [wizard.steps.characterReferences.form.inputs.phoneNumber.id]:
+          draftApplication.characterReferences && draftApplication.characterReferences[0] ? draftApplication.characterReferences[0].phoneNumber : null,
```

Comment:
dual check for characterReferences then accessing the 1st index needed when we do not have a draft application. 

accessing charRef[0] index would result in an exception.

Reply:
NITPICK: Same thing but a little less code:

```
[wizard.steps.characterReferences.form.inputs.firstName.id]:
  draftApplication.characterReferences?.[0]?.firstName ?? null,
```

Would empty string be better for wizard data?

```
[wizard.steps.characterReferences.form.inputs.firstName.id]:
  draftApplication.characterReferences?.[0]?.firstName ?? "",
```

Reply:
100%